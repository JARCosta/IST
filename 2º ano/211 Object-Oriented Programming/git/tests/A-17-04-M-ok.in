3
4
1
7
4
M1
SAL
10
10
4
M1
SAL
10
20
0
4
1
7
4
M2
VIDRO
10
10
0
4
3
7
4
M1
SAL
10
30
4
M2
VIDRO
10
40
3
M1
10
SAL
2
3
M2
12
SAL
2
3
S1
23
SAL
4
4
M1
VIDRO
10
10
0
6
5
M1
5
M2
5
S1
0
0
