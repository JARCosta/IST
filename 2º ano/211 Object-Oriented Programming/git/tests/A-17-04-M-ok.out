Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Data actual: 0
Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Número de dias a avançar: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Número de dias a avançar: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Número de dias a avançar: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|SAL|10|100|1
COMPRA|1|M1|SAL|20|200|1
COMPRA|3|M1|SAL|30|300|5
COMPRA|8|M1|VIDRO|10|100|5
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: COMPRA|2|M2|VIDRO|10|100|2
COMPRA|4|M2|VIDRO|40|400|5
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 